src/pudl_archiver/orchestrator.py

```diff
@@ -167,7 +167,7 @@ async def _initialize(self):
         self.deposition_files = {f.filename: f for f in self.new_deposition.files}
         self.changes: list[_DepositionChange] = []
 
-    async def run(self) -> RunSummary | Unchanged:
+    async def run(self) -> RunSummary:
```

Comment:
Got rid of `Unchanged` since that seemed to be adding more complexity than it was worth.